src/js/disability-benefits/components/Notification.jsx

```diff
@@ -0,0 +1,34 @@
+import React from 'react';
+import classNames from 'classnames';
+
+export default function Notification({ title, body, onClose, type }) {
```

Comment:
There is a common component `AlertBox`, should we be using that? It looks pretty similar but has a few differences in element nesting